pyproject.toml

```diff
@@ -5,4 +5,5 @@ description = "Add your description here"
 requires-python = ">=3.12"
 dependencies = [
     "seaborn>=0.13.2",
+    "jsonschema>=4.23.0",
```

Comment:
Also, I wonder if we should not split the python project for the implementation from the one we use for plots.

Then we could encapsulte lockfiles and configuration files in each implementations folders.

Reply:
I thought about that, but I figured I'd delay that until if and when we run into conflicts. But I'm fine making a separate environment if that feels cleaner.

Reply:
I did on a seperate PR for node: https://github.com/sourcemeta-research/jsonschema-benchmark/pull/20

So nw :)

Reply:
Made those changes here too.